server/src/test/java/org/elasticsearch/index/IndexServiceTests.java

```diff
@@ -367,70 +361,6 @@ public void testRescheduleAsyncFsync() throws Exception {
         assertNotNull(indexService.getFsyncTask());
     }
 
-    public void testAsyncTranslogTrimActuallyWorks() throws Exception {
```

Comment:
why remove this test (and the one below)? We have not removed this functionality and still need it, no?

Reply:
restored in https://github.com/elastic/elasticsearch/pull/51417/commits/82255aae678c57e1b0ab4c67d7e21be7266cfb7f.